TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Info.java

```diff
@@ -0,0 +1,9 @@
+package org.firstinspires.ftc.teamcode;
+
+import org.firstinspires.ftc.teamcode.gamelayout.Alliance;
+
+public class Info {
+
+    public static Alliance alliance; // set at start
```

Comment:
why is this not set at start?

Reply:
its need to happen in the init of auto because there is no way to get allaince

Reply:
yes but then why isn't it initialized somwhere else then?

Reply:
because the variable in use in multiply places, so something will init and the other things will use

Reply:
how does this get init when used?